Tests/SR25519/SNBIP39SeedTests.m

```diff
@@ -7,8 +7,11 @@
 //
 
 #import <XCTest/XCTest.h>
-@import NovaCrypto;
+#import "SNBIP39SeedCreator.h"
 #import "IRBIP39TestData+Load.h"
+#if __has_include(<NovaCrypto/NSData+Hex.h>)
+#import "NovaCrypto/NSData+Hex.h"
```

Comment:
Looks like we check one import for include but use a different one with "" instead of <>